### PR TITLE
reworked for sync

### PR DESCRIPTION
**Pull request by DrewWeth**
you guys were using documentFragment methods that didnt exist. noobs. reworked so that it works with sync.

production still hangs with any request on :3001

### CODE REVIEW

**Comment by DrewWeth** [comment]
@CBXZero @ryan-endacott , lemme know what you guys think.

**Comment by ryan-endacott** [comment]
Are we just moving the chat demo to this repo then?  I feel like we should keep it as a separate repo. 

I didn't test any of the code or the demo, but it looks good from a high level :+1: 

Only minor nitpick is that you should remove all the logging in the library, or at least put it behind a debug flag.  

Also, I can't believe you gave me fragment code that didn't work @bradcypert :stuck_out_tongue:

**Comment by Callie-Sylph** [comment]
Why is application getting passed to connect? I don't remember it being used there and don't see anywhere that it is used (though I'm currently on mobile and it won't let me see the whole file from here)

File: ease.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+  }
+  else{
+    this.baseUrl = urls.localhost;
+    this.syncUrl = urls.localSync;
+  }
+
   this.username = username;
   this.appName = appName;
   this.appToken = appToken;
+  this.prodBoolean = prodBoolean;
```
Comment (frankmariette) [inline ease.js:21]: This isn't needed. It's never used anywhere in the file.

File: ease.js [hunk 2]
```diff
@@ -39,6 +53,7 @@ function Ease(username, appName, appToken) {
       path : path,
       data : data
     };
+    console.log(dataToSend);
```
Comment (frankmariette) [inline ease.js:56]: Shouldn't have console logs of the data.

File: ease.js [hunk 3]
```diff
@@ -73,7 +88,7 @@ function Ease(username, appName, appToken) {
         appName: application,
         authorization: this.appToken
     };
-
+    console.log("Data to send: " + JSON.stringify(dataToSend))
```
Comment (frankmariette) [inline ease.js:91]: Same as above. No need to log this info.

File: ease.js [hunk 4]
```diff
@@ -65,7 +80,7 @@ function Ease(username, appName, appToken) {
 
   this.subscribe = function(application) {
     if(this.conn == undefined) {
-      this.connect();
+      this.connect(application);
```
Comment (frankmariette) [inline ease.js:83]: application can be referred to with `this.appName`. No need to have application passed in as it is redundant.

**Comment by Callie-Sylph** [comment]
Thank you for the cleanup check on my javascript @karrde00

File: ease.js [hunk 5]
```diff
@@ -1,14 +1,28 @@
-function Ease(username, appName, appToken) {
+function Ease(username, appName, appToken, prodBoolean) {
   var urls = {
-    localhost: 'localhost:3000/',
-    prod: 'ease-62q56ueo.cloudapp.net:3001'
+    localhost: 'localhost:3001',
```
Comment (ryan-endacott) [inline ease.js:3]: change urls

**Comment by DrewWeth** [comment]
Capstone meeting. I checked that old example and new example work with refreshing and sync implementation.

**Pull request merged**